packages/app/src/systems/Network/services/network.ts

```diff
@@ -58,8 +58,6 @@ export class NetworkService {
   }
 
   static async addNetwork(input: NetworkInputs['addNetwork']) {
-    // biome-ignore lint/complexity/noThisInStatic: <explanation>
```

Comment:
Removing this would stop checking if the added network is valid and compatible with the MIN_NODE_VERSION required by the wallet. Wouldn't it make more sense to create a second paremeter (e.g. `skipValidation`) so that only `addDefaultNetworks` works regardless of validation?

To make more sense with the name we could merge `addNetwork` and `validateAddNetwork` into one method, and make validation default true but optional, disabled only for the defaultNetworks call.

Reply:
It's better to keep this method with a single responsibility, isn't it?
We're still validating the networks, but now the methods are called when needed.

For example, when we're adding the default networks, I'm not validating since it's from our end.
But in the other flows (which are only 2), I've added the validation accordingly.

Take a look here

https://github.com/FuelLabs/fuels-wallet/blob/26e4e14ae7f09ac2147627ed90a8a67d88520717/packages/app/src/systems/DApp/machines/addNetworkRequestMachine.tsx#L123

Reply:
That way we keep the validation logic with a single responsibility, as well we keep the reusability of the original `addNetwork`.